pkg/kubectl/cmd/describe.go

```diff
@@ -39,36 +40,39 @@ type DescribeOptions struct {
 	Recursive bool
 }
 
-const (
-	describe_long = `Show details of a specific resource or group of resources.
+var (
+	describe_long = dedent.Dedent(`
+	Show details of a specific resource or group of resources.
```

Comment:
if we are nit-picking I would indent this one extra level.  But it's minor and not worth re-piushing unless there are other things to change